script/script.go

```diff
@@ -909,16 +911,11 @@ func getPathParam(f filters.FilterContext) func(*lua.LState) int {
 
 func unsupported(message string) func(*lua.LState) int {
 	return func(s *lua.LState) int {
-		s.RaiseError(message)
+		s.RaiseError(message+"%s", "")
```

Comment:
Yeah ugly hack but not possible to fix it because they have a format string and optional args, but vet requires now to have the optional parameter passed.